### PR TITLE
[151527583] Remove 4.4, move to alpine image, and use oauth2_proxy 2.2.0

### PR DESCRIPTION
**Pull request by khiemlam**
https://www.pivotaltracker.com/story/show/151527583

This is already been pushed up kibana-testing.  Please QA that app before approving this PR.  Once this merged to master, I will push it to kibana-testing.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,49 +1,45 @@
-FROM quay.io/aptible/ubuntu:14.04
+FROM alpine:3.6
 
 # Install NGiNX.
-RUN apt-get update && \
-    apt-get install -y software-properties-common \
-    python-software-properties && \
-    add-apt-repository -y ppa:nginx/stable && apt-get update && \
-    apt-get -y install curl ucspi-tcp apache2-utils ruby
+RUN apk update && \
```
Comment (EmergentBehavior) [inline Dockerfile:4]: I would look at some of our CI Dockerfiles and try to combine all your `RUN` statements together to help minimize image size.

**Review by EmergentBehavior — APPROVED** [review]
QA'd a bit on testing and it worked for me. Auto-refreshed and pulled up query I wanted.

Consider combining RUN statements together for smaller image. See our CI Dockerfiles. 

Worth noting that I saw this error on bengal-sandbox from the dash (repeating again and again):
```
September 28th 2017, 16:03:40.507 | ./run.sh: line 3: S3_TARBALL: Error: environment variable S3_TARBALL should be set.
-- | --
```

File: Dockerfile (hunk 1 above)
Comment (khiemlam) [reply Dockerfile:4]: Yeah, I combine as much as reasonably possible for that setup step.  Trying to stay in sweet spot of minimizing the layers while making it easy to pull something from cache if needed.

**Comment by khiemlam** [comment]
@EmergentBehavior Cool, I'll merge this and then looking into the Bengal S3 tarball issue

**Pull request merged**